### ISSUE TITLE
Playback commands are not in the PIP package

### ISSUE THREAD

**Issue by gawaineo** [plamere/spotipy#231]
When I install spotipy via PIP and try to run a playback command, they are not found in the package.
Commands such as **start_playback, next_track, previous_track**. I also checked the site-packages folder inside my virtual environment and in the source code those methods are not found.
```
Traceback (most recent call last):
  File "spot.py", line 19, in <module>
    spotify.previous_track()
AttributeError: 'Spotify' object has no attribute 'previous_track'
```

When I clone the repo, and install via setup.py install, the playback commands work fine, but it would be nice to have it work when you install via PIP.

**Comment by ritiek** [comment]
+1 on this. Although #211 talks about the same problem.

**Comment by stephanebruckert** [comment]
This should be fixed with version 2.5.0, just do:

    pip install spotipy --upgrade
